hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/api/db.json

```diff
@@ -6686,5 +6686,73 @@
         ]
       }
     ]
+  },
+  "dirdeletePending": {
+    "lastKey": "dir11",
```

Comment:
I think this is not necessary.
```suggestion
```

Reply:
@dombizita  Pushed changes in latest commit